cob_voltage_control/common/src/cob_voltage_control_common.cpp

```diff
@@ -28,9 +28,9 @@ class cob_voltage_control_data
     int in_phidget_current;
 
     //output data
-    cob_msgs::PowerBoardState out_pub_em_stop_state_;
+    cob_msgs::PowerBoardState out_pub_relayboard_state;
```

Comment:
Should we call it `out_pub_powerboard_state` to be consistent with the message type and to remove any (obsolete/missleading) relation to relayboard?